.github/workflows/release.yml

```diff
@@ -103,6 +104,24 @@ jobs:
         with:
           python-version: 3.8
 
+      - name: Cache cargo plugins
```

Comment:
does this really only cache cargo plugins?
doesnt this cache more cargo related stuff?